ui/src/app/base/components/Col/Col.js

```diff
@@ -0,0 +1,73 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+const colRange = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9,
+  10,
+  11,
+  12,
+  "1",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "10",
+  "11",
+  "12"
+];
```

Comment:
This could be more succinct, but I'm not honestly sure if it is better!
```javascript
let colRange = Array.from({length: 13}, (v, i ) => i);
colRange = [...colRange, ...colRange.map(i => `${i}`)]
```

Reply:
I thought about this too, I'm not sure if gains us anything except for not having strange Prettier formatting.